bin/teamshow

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/python
+
+"""
+    Script to show LAG and LAG member status in a summary view
+    Example of the output:
+    acsadmin@CO4SCH03010AALF:~$ teamshow
+"""
+
+import json
+import re
+import subprocess
+import sys
+from tabulate import tabulate
+
+class Teamshow(object):
+    def __init__(self):
+        self.ipraw = None
+        self.teams = []
+        self.teamsraw = {}
+        self.summary = {}
+        self.err = None
+
+    def get_ip(self):
+        """
+            Command: 'ip -o link show'
+        """
+        ip_cmd = 'ip -o link show'
+        p = subprocess.Popen(ip_cmd, stdout=subprocess.PIPE, shell=True)
+        (output, err) = p.communicate()
+        rc = p.wait()
+        if rc == 0:
+            self.ipraw = output
+        else:
+             self.err = err
+
+    def parse_ip(self):
+        re_ip = "\d+:\s(PortChannel\d+):"
+        for line in self.ipraw.split('\n'):
+            result = re.match(re_ip, line)
+            if result is not None:
+                self.teams.append(result.group(1))
+
+    def get_teamdctl(self):
+        """
+            Command: 'teamdctl <teamdevname> state dump'
+        """
+        for team in self.teams:
+            teamdctl_cmd = 'teamdctl ' + team + ' state dump'
+            p = subprocess.Popen(teamdctl_cmd, stdout=subprocess.PIPE, shell=True)
+            (output, err) = p.communicate()
+            rc = p.wait()
+            if rc == 0:
+                self.teamsraw[team] = output
+            else:
+                self.err = err
+
+    def parse_teamdctl(self):
+        for team in self.teams:
+            info = {}
+            json_info = json.loads(self.teamsraw[team])
+            info['protocol'] = json_info['setup']['runner_name'].upper()
+            info['protocol'] += '(A)' if json_info['runner']['active'] else '(I)'
+            info['ports'] = ""
+            for port in json_info['ports']:
+                info['ports'] += port
+                info['ports'] += '(S)' if json_info['ports'][port]['runner']['selected'] else '(D)'
+                info['ports'] += ' '
+            self.summary[team] = info
+
+    def display_summary(self):
+        print "Flags: A - active, I - inactive, S - selected, D - deselected"
+        header = ['Team Dev', 'Protocol', 'Ports']
+        output = []
+        for team in self.summary:
+            output.append([team, self.summary[team]['protocol'], self.summary[team]['ports']])
+        print tabulate(output, header)
+
+def main():
+    try:
+        team = Teamshow()
+        team.get_ip()
+        team.parse_ip()
```

Comment:
in order to use minigraph, i'll also need this https://github.com/Azure/sonic-utilities/pull/6 to get the alias_map.json file.